### PR TITLE
296 fix benchmark build errors

### LINKED ISSUE #296: Fix benchmark build errors

### PR DESCRIPTION
**Pull request by archeoss**
Resolves #296

### CODE REVIEW

File: src/benchmark/bin.rs [hunk 1]
```diff
@@ -73,62 +73,63 @@ async fn start_app() {
...
-            );
-            if percent % 50 == 0 {
-                println!();
-            }
-        }
-        prev_p = percent;
-        if futures_pool.len() < futures_limit {
-            if let Some((key, data)) = generator.next() {
+    {
+        let arc_writer = Arc::new(&writer);
```
Comment (ikopylov) [inline src/benchmark/bin.rs:82]: Looks like `arc_writer` is not needed. You can use `writer` directly

**Comment by ikopylov** [comment]
Please, add to `build-linux` job (https://github.com/qoollo/pearl/blob/master/.github/workflows/build.yml#L10) a new step, that builds benchmark (only benchmark `--bin benchmark`)

**Comment by archeoss** [comment]
Current dependencies on/for this PR:
* `master`
  * **PR #297** <a href="https://app.graphite.dev/github/pr/qoollo/pearl/297?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/qoollo/pearl/297?utm_source=stack-comment).

**Comment by ikopylov** [comment]
I don't see the benchmark compilation in the build logs (https://github.com/qoollo/pearl/actions/runs/7231822100/job/19705418934?pr=297). Please check whether the `benchmark` is built

**Comment by archeoss** [comment]
@ikopylov It's definitely builds: https://github.com/qoollo/pearl/actions/runs/7245092090/job/19734481484?pr=297

**Review by ikopylov — APPROVED** [review]
LGTM

**Pull request merged**